monkey/monkey_island/cc/next_ui/src/_components/app-nav/app-bar/AppBar.tsx

```diff
@@ -1,20 +1,41 @@
 'use client';
+import React from 'react';
 import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
 import Container from '@mui/material/Container';
+import HomeIcon from '@mui/icons-material/Home';
+import MenuItem from '@mui/material/MenuItem';
+import SvgIcon from '@mui/material/SvgIcon';
 import Toolbar from '@mui/material/Toolbar';
-import classes from './appBar.module.scss';
-import React from 'react';
-import AppMenu from '@/_components/app-nav/app-menu/AppMenu';
-import AppLogo from '@/_components/app-nav/app-logo/AppLogo';
+import Typography from '@mui/material/Typography';
 import AppAvatar from '@/_components/app-nav/app-avatar/AppAvatar';
-import Box from '@mui/material/Box';
 import AppDrawerOpener from '@/_components/app-nav/app-drawer-opener/AppDrawerOpener';
+import AppMenu from '@/_components/app-nav/app-menu/AppMenu';
+import MonkeyHeadSvg from '@/assets/svg-components/MonkeyHead.svg';
 import useSmallScreenCheck from '@/hooks/useSmallScreenCheck';
-import { ThemeMode } from '@/_components/theme-mode/ThemeMode';
+import { PATHS } from '@/constants/paths.constants';
+import { useRouter } from 'next/navigation';
+import classes from './appBar.module.scss';
+
+const AboutLink = { path: PATHS.ABOUT, label: 'About', icon: <HomeIcon /> };
 
-const MonkeyAppBar = ({ setIsDrawerOpen = null }: { setIsDrawerOpen: any }) => {
+export interface MenuProps {
+    onClose?: () => void;
+}
+
+const MonkeyAppBar = (
```

Comment:
Component should be renamed to `MonkeyNavSx` or something similar to indicate that this component defines navigation for small screens.